File: 백준/Bronze/2576. 홀수/홀수.py
sum = 0
min = 101
for i in range(7):
    n = int(input())
    if n % 2 != 0:
        sum += n
        if min > n:
            min = n


if min == 101:
    print(-1)
else:
    print(sum)
    print(min)

File: 백준/Bronze/2576. 홀수/홀수.c
#include <stdio.h>
#include <stdlib.h>
#include <string.h>
#include <math.h>
#include <ctype.h>
#include <limits.h>

#define TRUE 1 
#define FALSE 0

int main() {
    int min = INT_MAX;
    int sum = 0;

    for(int i = 0; i < 7; i++){
        int num;
        scanf("%d", &num);
        if(num % 2 == 1){
            sum += num;
            if(num < min){
                min = num;
            }
        }
    }

    if(sum == 0){
        printf("-1\n");
    } else {
        printf("%d\n%d\n", sum, min);
    }

    return 0;
}